content/en/platform/corda/5.2/application-networks/upgrading/_index.md

```diff
@@ -11,27 +11,32 @@ menu:
 
 # Upgrading an Application Network
 
-If an administrator [re-registers the MGM]({{< relref "../creating/mgm/reregister.md" >}}) or [upgrades a Corda cluster from 5.0 to 5.1]({{< relref "../../deploying-operating/deployment/upgrading/_index.md" >}}), you should upgrade the member {{< tooltip >}}CPIs{{< /tooltip >}} of your application network. The `MemberInfo` platform version is set at member registration time and so upgrading members ensures that this value reflects the current version. Upgrading members enables the following:
+If an administrator [re-registers the MGM]({{< relref "../creating/mgm/reregister.md" >}}) or [upgrades a Corda cluster from 5.1 to 5.2]({{< relref "../../deploying-operating/deployment/upgrading/_index.md" >}}), you should upgrade the MGM and the member {{< tooltip >}}CPIs{{< /tooltip >}} of your application network. The `MemberInfo` platform version is set at member registration time and so upgrading members ensures that this value reflects the current version. Upgrading members enables the following:
 
 * Members see the latest `MemberInfo` of the MGM.
```

Comment:
Important to note that this will be only true if they really re-register the MGM before upgrading their CPIs. Without MGM re-registration and re-exporting the new group policy file, they will think that the MGM is on older platform version.